script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9_1.txt

```diff
@@ -93,18 +93,18 @@ However.
 It is impossible for a human to cross those two steps.
 <Page26>
 "――――――"
-The room to be able to scream,
-and the ability to feel pain was already frozen in place.
-Was he aware of my plan, or was it just an automatic interception?
-There was a countless number of ice thorns growing around Vlov to greet the attacker.
+My ability to scream,//悲鳴をあげるだけの余裕も、
+and my capacity to feel pain were already frozen solid.//苦痛を感じるだけの感覚も、既に凍っていた。
+Did he notice me, or was it simply an autonomic counterattack?
+A countless number of ice thorns had sprouted around Vlov to greet any would-be attacker.
 <Page27>
 "―――As the fool you are, you chose the path of safety."
 A voice of disappointment reaches me.
 Safety. Safety. Safety, huh.
 I was thinking about a way to save myself.
 Out of concern for me, Arcueid threw my debris a second slower compared to the other ones.
 ...This was a failure. This was beyond foolish of me.
-Something like this was nothing like a surprise attack.
+This was nowhere close to a surprise attack.
 <Page28>
 There is no need to wait until I bleed to death.
-The pain of the cold wave and my disappointment in myself has brought my heart to a complete stop.
+My suffering from being afflicted by Vlov's cold wave, combined with my own disappointment in myself, has brought my heart to a complete stop.//罹 seems a little disease-y ヴローヴに罹った寒波と、己への失望から、俺の心臓は完全に停止していた。
```

Comment:
First clause is slightly wrong.
> ヴローヴに罹った寒波
The cold wave that afflicted Vlov

It's Vlov's affliction, not Shiki's

Reply:
Whoops, brain must have been fried from staring at this one too long